lms/djangoapps/mobile_api/video_outlines/tests.py

```diff
@@ -114,6 +126,86 @@ def _create_video_with_subs(self):
         )
 
 
+class TestEmptyCourseVideoSummaryList(MobileAPITestCase):
+    """
+    Tests /api/mobile/v0.5/video_outlines/courses/{course_id} with no course set
+    """
+    REVERSE_INFO = {'name': 'video-summary-list', 'params': ['course_id']}
+
+    def test_chapter_is_none(self):
+        """
+        Tests when there is no chapter under course, and video under course
+        """
+        self.login_and_enroll()
+        ItemFactory.create(
+            parent_location=self.course.location,
+            category="video",
+            display_name=u"test factory video omega \u03a9",
+        )
+        course_outline = self.api_response().data
+        self.assertEqual(len(course_outline), 1)
+        section_url = course_outline[0]["section_url"]
+        unit_url = course_outline[0]["unit_url"]
+        self.assertRegexpMatches(section_url, r'courseware$')
+        self.assertTrue(section_url)
+        self.assertTrue(unit_url)
```

Comment:
Ah I see now. I included that in there because of the case where they could be None or "", but now that there is a regex to validate it, those two cases are already account for.